docs/src/main/asciidoc/opentelemetry-dev-services.adoc

```diff
@@ -0,0 +1,41 @@
+////
```

Comment:
Please mutually link this guide with https://quarkus.io/version/main/guides/opentelemetry